src/interfaces/alteration-interfaces/business-snapshot-interface.ts

```diff
@@ -1,21 +1,22 @@
 import {
-  AuthInformationIF, BusinessInformationIF, GetOrgPersonIF, IncorporationAddressIf,
-  NameTranslationIF, ShareStructureIF, ResolutionsIF, BaseAddressObjIF
+  AddressesIF, AuthInformationIF, BusinessInformationIF, GetOrgPersonIF,
+  NameTranslationIF, ShareStructureIF, ResolutionsIF
 } from '@/interfaces'
 
 /** Entity snapshot object. */
 export interface EntitySnapshotIF {
   businessInfo: BusinessInformationIF
   authInfo: AuthInformationIF
   orgPersons: GetOrgPersonIF[]
+  addresses: AddressesIF
 
   // Properties exclusive to Companies
-  incorporationAddress?: IncorporationAddressIf
+  // incorporationAddress?: AddressesIF
```

Comment:
Obsolete or for future use? Also line 21.

Reply:
Indeed, will clean this up!

Reply:
Updated